### PR TITLE
Changing Test System

### LINKED ISSUE #18: Migrating to GitHub Actions
#### Description
Migrating from Travis/Appveyor to GitHub Actions

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#18 

Changes:
+ `travis.yml` removed
+ `appveyor.yml` removed
+ `test.yml` (Github actions test workflow) added

### CODE REVIEW

**Comment by sadrasabouri** [comment]
Hi @sepandhaghighi .
Please review this PR and sign out from Appveyor avoiding it's failure.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+https://soundcloud.com/noisetrappy-0fficialhttps://soundcloud.com/noisetrappy-0fficial# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
```
Comment (sepandhaghighi) [inline .github/workflows/test.yml]: What??!!!

Comment (sepandhaghighi) [reply .github/workflows/test.yml]: also, remove `and lint`

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

**Comment by sepandhaghighi** [comment]
> Hi @sepandhaghighi .
> Please review this PR and sign out from Appveyor avoiding it's failure.

Done ;-)

**Comment by sepandhaghighi** [comment]
@sadrasabouri 
Please change the target branch to `master`, `dev` is stale we should remove it.
Take a look at [Contribution](https://github.com/ECSIM/dbfc-dataset/blob/master/.github/CONTRIBUTING.md)

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]
Please update `Notebooks` folder  badges

File: Notebooks/README.md [hunk 2]
```diff
@@ -7,18 +7,14 @@
 
 <table style="border-collapse: collapse;">
 	<tr>
-		<td align="center"><b>Branch</b></td>
-		<td align="center"><b>Travis</b></td>
-		<td align="center"><b>Appveyor</b></td>
+		<td align="center">CI</td>
+		<td align="center"><img src="https://github.com/ECSIM/dbfc-dataset/workflows/CI/badge.svg?branch=master"></td>
+		<td align="center"><img src="https://github.com/ECSIM/dbfc-dataset/workflows/CI/badge.svg?branch=dev"></td>
```
Comment (sepandhaghighi) [inline Notebooks/README.md]: There is no `dev` branch
Please remove this line

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: Notebooks/README.md (hunk 2 above)
Comment (sadrasabouri) [reply Notebooks/README.md]: Excuse me for this careless commit.

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
LGTM 💯

**Pull request merged**